spec/actions/inject_into_file_spec.rb

```diff
@@ -70,6 +70,22 @@ def file
       expect(File.read(file)).to eq("__start__\nREADME\nmore content\n__end__\n")
     end
 
+    it "does not attempt to change the file if it doesn't exist" do
+      invoker.inject_into_file "idontexist", :before => "something" do
+        "any content"
+      end rescue nil
```

Comment:
Maybe instead of rescue here we can merge both tests and assert that the file was not created in the test below?